h5py/tests/test_h5d_direct_chunk.py

```diff
@@ -186,3 +186,75 @@ def test_fail_buffer_not_contiguous(self, writable_file):
         out = array[:, :, ::2]  # Array is not contiguous
         with pytest.raises(ValueError):
             dataset.id.read_direct_chunk((0, 0), out=out)
+
+
+@ut.skipIf(h5py.version.hdf5_version_tuple < (1, 14, 0),
+           "read_multi requires HDF5 >= 1.14.0")
+class TestReadMulti(TestCase):
+    def test_read_multi_one_dataset(self):
+        filename = self.mktemp().encode()
+        with h5py.File(filename, "w") as filehandle:
+            shape = (10, 10, 10)
+            dt = numpy.int32
+
+            # Write data
+            data_in = numpy.reshape(numpy.arange(numpy.prod(shape)), shape)
+            dataset = filehandle.create_dataset("data", shape,
+                                                dtype=dt, data=data_in)
+
+            self.assertTrue(numpy.array_equal(data_in, dataset[...]))
```

Comment:
```suggestion
            numpy.testing.assert_array_equal(data_in, dataset[...])
```

And throughout. Specific assert methods like this can give you more useful information if the thing they're checking fails, whereas `assertTrue` can only tell you it was not true.  E.g.

```
AssertionError: 
Arrays are not equal

(shapes (5,), (6,) mismatch)
 x: array([0, 1, 2, 3, 4])
 y: array([0, 1, 2, 3, 4, 5])
```